core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -467,7 +481,25 @@ public LocalExecutionPlanner(
         this.joinCompiler = requireNonNull(joinCompiler, "joinCompiler is null");
         this.operatorFactories = requireNonNull(operatorFactories, "operatorFactories is null");
         this.orderingCompiler = requireNonNull(orderingCompiler, "orderingCompiler is null");
-        this.dynamicFilterConfig = requireNonNull(dynamicFilterConfig, "dynamicFilterConfig is null");
+        requireNonNull(dynamicFilterConfig, "dynamicFilterConfig is null");
+        this.largeBroadcastMaxDistinctValuesPerDriver = dynamicFilterConfig.getLargeBroadcastMaxDistinctValuesPerDriver();
+        this.smallBroadcastMaxDistinctValuesPerDriver = dynamicFilterConfig.getSmallBroadcastMaxDistinctValuesPerDriver();
+        this.smallPartitionedMaxDistinctValuesPerDriver = dynamicFilterConfig.getSmallPartitionedMaxDistinctValuesPerDriver();
+        this.largeBroadcastMaxSizePerDriver = dynamicFilterConfig.getLargeBroadcastMaxSizePerDriver();
+        this.largePartitionedMaxSizePerDriver = dynamicFilterConfig.getLargePartitionedMaxSizePerDriver();
+        this.smallBroadcastMaxSizePerDriver = dynamicFilterConfig.getSmallBroadcastMaxSizePerDriver();
+        this.smallPartitionedMaxSizePerDriver = dynamicFilterConfig.getSmallPartitionedMaxSizePerDriver();
+        this.largeBroadcastRangeRowLimitPerDriver = dynamicFilterConfig.getLargeBroadcastRangeRowLimitPerDriver();
+        this.largePartitionedRangeRowLimitPerDriver = dynamicFilterConfig.getLargePartitionedRangeRowLimitPerDriver();
+        this.smallBroadcastRangeRowLimitPerDriver = dynamicFilterConfig.getSmallBroadcastRangeRowLimitPerDriver();
+        this.smallPartitionedRangeRowLimitPerDriver = dynamicFilterConfig.getSmallPartitionedRangeRowLimitPerDriver();
+        this.largeBroadcastMaxSizePerOperator = dynamicFilterConfig.getLargeBroadcastMaxSizePerOperator();
+        this.largePartitionedMaxSizePerOperator = dynamicFilterConfig.getLargePartitionedMaxSizePerOperator();
+        this.smallBroadcastMaxSizePerOperator = dynamicFilterConfig.getSmallBroadcastMaxSizePerOperator();
+        this.smallPartitionedMaxSizePerOperator = dynamicFilterConfig.getSmallPartitionedMaxSizePerOperator();
+
+
+        this.largePartitionedMaxDistinctValuesPerDriver = dynamicFilterConfig.getLargePartitionedMaxDistinctValuesPerDriver();
```

Comment:
Is this gap in purpose?

Reply:
No. Thanks for catching that. Removed the empty lines.